### ISSUE TITLE
[Accessibility] The "Table.ItemColumnHeaderItems" property of MonthCalendar should not be empty in the inspect tool

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#4977]
- .NET Core Version: .NET SDK 6.0.100-Preview.5.21273.2

- Have you experienced this same bug with .NET Framework?:
No, the property does not exist in the Framework.

**Problem description:**
The "Table.ItemColumnHeaderItems" property of MonthCalendar should not be empty in the inspect tool, it should be pointed to corresponding "Mon/Tue/Wed/Thu/Fri/Sat/Sun".

**Minimal repro:**

1. Create a .NET Core project with a MonthCalendar control.
2. Build and run the core project.
3. Observe the "Table.ItemColumnHeaderItems" property of MonthCalendar in the Inspect tool on UI Automation mode.

Or

1. Open the attached project.
2. Build and run the core project.
3. Observe the "Table.ItemColumnHeaderItems" property of MonthCalendar in the Inspect tool on UI Automation mode.
[WinFormsApp8.zip](https://github.com/dotnet/winforms/files/6545580/WinFormsApp8.zip)

**Expected behavior:**
Inspect result: The "Table.ItemColumnHeaderItems" property of MonthCalendar should not be empty. Following screenshot from latest private testing confirmed with Vladimir.
![image](https://user-images.githubusercontent.com/45864649/119635374-34566f00-be46-11eb-9046-4b6dd9f08420.png)
Narrator result: The column header should be read to corresponding string "Column Header Mon/Tue/Wed/Thu/Fri/Sat/Sun".
![image](https://user-images.githubusercontent.com/45864649/119635724-8a2b1700-be46-11eb-8a9f-269d7547f595.png)

**Actual Behavior:**
Inspect result: The "Table.ItemColumnHeaderItems" property of MonthCalendar is empty.
![image](https://user-images.githubusercontent.com/45864649/119635893-b2b31100-be46-11eb-9083-cb7b88357bcc.png)
Narrator result: The column header is not read when the date is switched from left to right.
![image](https://user-images.githubusercontent.com/45864649/119635764-92835200-be46-11eb-8463-2900ed3142cf.png)

**Comment by Olina-Zhang** [comment]
This issue was found the PR: https://github.com/dotnet/winforms/pull/4758 merged

**Comment by Jessie-Zhang01** [comment]
Verified the issue with .Net SDK 6.0.100-preview.6.21278.6, this issue is fixed. Please see following screenshot:
Inspect result:
![image](https://user-images.githubusercontent.com/45864649/120135476-ebfece80-c202-11eb-8a0f-5438b0c72cf0.png)
Narrator result:
![image](https://user-images.githubusercontent.com/45864649/120135492-f4efa000-c202-11eb-8d39-96685794e41e.png)
